api/docker-compose.yml

```diff
@@ -1,9 +1,7 @@
-version: '3.2'
-
 services:
     db:
         container_name: api-test-db
-        image: postgres:14.5
+        image: postgres:14.11
```

Comment:
Thoughts on moving to an alpine image? [14.11-alpine](https://hub.docker.com/layers/library/postgres/14.11-alpine/images/sha256-ac4ca373551069f5cceb157bc4ba7fefc67069f3cf7dfb96fa3fbfece9888d5f?context=explore)

14.11 still has vulnerabilities - https://hub.docker.com/layers/library/postgres/14.11/images/sha256-a28005f051c26406f8e718e411a1068462a297dae049cddb513194d36dd8f29f?context=explore